config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  root 'restaurants#index'
+  root 'home#index'
 
   resources :restaurants, only: [:index, :show]
+  resources :genres, only: [:index]
```

Comment:
I would use an `api/v1` namespace for these endpoints

Reply:
I think that an API is a better fit for dedicated API (consumed by external apps), in this case you have to manage the API versions etc.
In this specific case I would rather be explicit and see where things go (as in the future you'll want to be able to add genres, edit them etc